File: WebshopServer/Dtos/UserDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebshopServer.Enums;

namespace WebshopServer.Dtos
{
    public class UserDto
    {
        public long Id { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Birthdate { get; set; }
        public string Address { get; set; }
    }
}

File: webshop-client/src/models/userDto.js
export function userResponseDto(data) {
  return {
    id: data.id,
    username: data.username,
    email: data.email,
    password: data.password,
    firstName: data.firstName,
    lastName: data.lastName,
    birthdate: data.birthdate,
    address: data.address,
    role: data.role,
    verificationStatus: data.verificationStatus,
  };
}
